### DISCUSSION TITLE
Windows defender identifies CHIA 20.0 as a trojan and quarantines the installed program.

### DISCUSSION THREAD

**Discussion by RobertGWR** [Chia-Network/chia-blockchain#16165] [Support]
Windows defender identifies CHIA 20.0 as a trojan and quarantines the installed program.
Can this be addressed? I had to allow the Trojan to continue farming.

**Comment by shaneo257** [comment]
hi, it is a false positive

**Comment by RobertGWR** [comment]
Will you be posting something in the thread that I started to let anyone else know why this false positive occurs and what to do about it? It after all should windows defender be active it quarenteens the Chia program and stops all farming activities.Robert
    On Sunday, August 27, 2023 at 06:16:04 p.m. MDT, shaneo257 ***@***.***> wrote:  
 
 


hi, it is a false positive

**Comment by BrandtH22** [comment]
This is a known issue and has been resolved with the latest microsoft defender update.

If you are still experiencing this issue please update your microsoft defender and reattempt :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
